spec/project/object_spec.rb

```diff
@@ -157,14 +157,10 @@ module ProjectSpecs
         group.files.should.be.empty
       end
 
-      it 'raises if it encounters an unknown attribute in a plist' do
+      it 'warns if it encounters an unknown attribute in a plist' do
+        STDERR.expects(:puts).times(1)
```

Comment:
can you change this to `UI.expects(:warn).with ...` to test that we actually warn with the expect string?